packages/sdk/src/fee/__tests__/FeeEstimator.spec.ts

```diff
@@ -0,0 +1,13 @@
+import { FeeEstimator } from "..";
+import { createMockPsbt } from "../utils";
+
+describe("FeeEstimator", () => {
+  test("should return an estimated network fee for P2SH-P2WPKH psbt when calculateNetworkFee is called", () => {
+    const feeEstimator = new FeeEstimator({
+      feeRate: 1,
```

Comment:
@iamcrazycoder not sure what's a regular fee rate? i can't seem to set decimals here. i've set it to 1 for this test.

Reply:
Fee rate in floating point aren't allowed by `psbt.setMaximumFeeRate` -- the reason why we don't allow it.